### DISCUSSION TITLE
qdtrack implementation based on YOLO-X detector

### DISCUSSION THREAD

**Discussion by laybebe** [ethvis/qd-track#140] [Q&A]
Thanks for your wonderful jobs! Recently, I have tried to make qdtrack implementation based on YOLO-X detector. However, I encountered difficulties. I sincerely expect the authors to release the code or provide help. Best wishes to you！

**Comment by epris** [comment]
Hi @laybebe, any news on that?

**Reply by laybebe** [reply]
Up to now, there is still no news.
